console/frontend/src/main/frontend/css/style.css

```diff
@@ -7051,3 +7051,27 @@ body.body-small .vote-icon {
 .dataTables_wrapper {
 	position: static;
 }
+/* datatable buttons fix, Selects all <a> */
+.dataTables_paginate a {
+  color: #2F4050 !important;
+}
+
+.dataTables_paginate a:hover{
+  background-color: #E9E9ED !important;
+  border: none !important;
+  transition: all 0.3s ease-in-out;
```

Comment:
Ik denk dat het niet verstandig is om een transitie te doen terwijl andere buttons met een hover effect dat ook niet hebben.